.evergreen/run-tests.sh

```diff
@@ -47,6 +47,12 @@ if test "$DOCKER_PRELOAD" != 1; then
   install_mlaunch_venv
 fi
 
+# Make sure cmake is installed (in case we need to install the libmongocrypt
+# helper)
+if test -n "$FLE"; then
```

Comment:
```suggestion
if [ "$FLE" = "helper" ]; then
```
Strictly speaking, we need cmake only if we use the helper gem. If we use the pre-compiled library, cmake is not needed.